frontend/dispatcher.lua

```diff
@@ -340,6 +342,8 @@ local dispatcher_menu_order = {
     "book_description",
     "book_cover",
 
+    "translate_page",
```

Comment:
(OK about having it here, can't find a better place in the Reader> actions menu.)